intera_examples/scripts/constrained_zeroG.py

```diff
@@ -24,16 +24,168 @@
 import intera_interface
 from intera_interface import CHECK_VERSION
 
+class ConstrainedZeroG(object):
+    """
+    Send interaction commands to place the robot in constrained zero-g mode
+    """
+
+    def __init__(self, args):
```

Comment:
This initialization method is long and complicated! Is there any way to break it up into smaller chunks that are easier to understand? 

How are failures inside of the initialization method handled? It looks like an error is issued, but that there is no other indication that the internal state of the object is invalid.